reference/features/synonyms.md

```diff
@@ -1,69 +1,85 @@
 # Synonyms
 
-To make your search more relevant, you can [create a list of synonyms](/reference/api/synonyms.md#update-synonyms).
+If multiple words have an equivalent meaning in your dataset, you can [create a list of synonyms](/reference/api/synonyms.md#update-synonyms). This will make your search results more relevant.
 
-If multiples words have an equivalent meaning in your dataset, you can decide to create a synonym list for these words. The search engine will give the same search results for any search with one of the associated words as a query.
+In general, **a search on a word will return the same results as a search on any of its synonyms**.
+There is one exception to this rule, [detailed below](#multi-word-phrases).
 
-Synonyms are considered to be the same.
-A search on a word or its synonym will return the same search result.
+## Normalization
 
-::: warning
-However, when a sentence is considered the synonym of another word or sentence, when searching the word or sentence will always be more relevant than its synonym. It will still be in the search results in the absence of a more relevant result.
-:::
+All synonyms are **lowercased** and **de-unicoded** during the indexing process.
 
-There are several ways to associate words with each other.
+#### Example
+
+Consider a situation where "Résumé" and "CV" are set as synonyms.
+
+```json
+{
+  "Résumé": ["CV"],
+  "CV": ["Résumé"]
+}
+```
 
-## The One-way association
+A search for "cv" would return any documents that contain the strings "Résumé", "resumé", or "resume", without being affected by case or accent marks.
 
-This makes it possible to determine that a word will be synonymous with another but not the other way around.
+## One-way Association
 
-example:
+Use this when you want one word to be synonymous with another, but not the other way around.
 
 ```
 phone => iphone
 ```
 
-By searching `phone` you will get all results containing `iphone` with the same relevance. However, if you search for `iphone`, documents containing `phone` will not be shown in your results.
+By searching `phone`, you will get all results containing `iphone` with the same relevance. However, if you search for `iphone`, documents containing `phone` will not be returned in your results.
```

Comment:
```
However, if you search for `iphone`, documents containing `phone` will not be returned in your results.
```
Does anybody want to test this and confirm that it's true? I'm worried that due to typo tolerance, querying `iPhone` might still return results that contain `phone`.

Can/should we think of a better example?

Reply:
The difference will be that without the synonyms documents with "phone" will be deprioritized behind documents with iPhones due to the type ranking rule. But if you add a synonym for "iPhone" => "phone", the documents with iPhone or phone will be classified without typo, so with the same relevancy.